janus_core/calculations/md.py

```diff
@@ -315,7 +334,7 @@ def __init__(  # pylint: disable=too-many-arguments,too-many-locals,too-many-sta
 
         self.minimize_kwargs = minimize_kwargs if minimize_kwargs else {}
         self.restart_files = []
-        self.dyn = None
+        self.dyn: Union[Langevin, VelocityVerlet, ASE_NPT]
```

Comment:
i think we have nph too

Reply:
`NPH` appears to be a derivative of `NPT`.

c.f. ll. 1137-1213